### PR TITLE
blog.twitter.com

### PR DESCRIPTION
**Pull request by SKEIDs**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [x] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

https://blog.twitter.com/ja_jp
 https://blog.twitter.com/ja_jp/topics/company/2022/world_suicide_prevention_day_event

#### If possible, a screenshot of a page or application should not be cropped too much. Otherwise, it is not always clear where the element is located

<details><summary>screenshot</summary>

</details><br/>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: SocialFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -163,6 +163,7 @@ confidenceman-movie.com#?#.share > ul > li.share_t:contains(Share)
 httpstatus.io#?#.content-section > div > section:has(> div.hero-body a[data-social-action="share"])
 httpstatus.io#?#.navbar-item:has(span:contains(Share))
 cmd4win.ru###right > div.moduletable:has(> div.custom > div[align="center"] > div#vk_groups)
+blog.twitter.com#?#.post > .post__share--icons:upward(1)
```
Comment (Yuki2718) [inline SocialFilter/sections/css_extended.txt:166]: specify tag on child i.e. `#?#.post > div.post__share--icons:upward(1)`.

File: SocialFilter/sections/specific.txt [hunk 2]
```diff
@@ -4451,6 +4451,7 @@ world-smartphones.com.ua##.socvid
 okulsoru.com##.tl-postShare
 51015kids.eu##.facebook_hidden
 blog.google,g-ek.com##.article-sticky-share
+blog.twitter.com##a[class^="sharing-popup-"]
```
Comment (Yuki2718) [inline SocialFilter/sections/specific.txt:4454]: `##.bl03-featured-masthead__share--icons` or `##div[class$="featured-masthead__share--icons"]`

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: SocialFilter/sections/css_extended.txt (hunk 1 above)
Comment (SKEIDs) [reply SocialFilter/sections/css_extended.txt:166]: ok

File: SocialFilter/sections/specific.txt (hunk 2 above)
Comment (SKEIDs) [reply SocialFilter/sections/specific.txt:4454]: ok

**Review by Yuki2718 — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**